spec/integration/transport/winrm_spec.rb

```diff
@@ -78,7 +78,7 @@ def stub_winrm_to_raise(klass, message)
 
       # The connection should fail immediately; this timeout helps ensure that
       # and avoids a hang
-      Timeout.timeout(3) do
+      Timeout.timeout(45) do
```

Comment:
These timeouts probably dont need to be this long. I think maybe the three seconds might have been just a little bit too slow to get the connect error for the closed port.